idaes/core/solvers/petsc.py

```diff
@@ -360,6 +357,19 @@ def _sub_problem_scaling_suffix(m, t_block):
             if c in m.scaling_factor:
                 t_block.scaling_factor[c] = m.scaling_factor[c]
 
+class PetscDAEResults(object):
+    """This class stores the results of ``petsc_dae_by_time_element()``, it has.
+    two attributes ``results`` and ``trajectory``.  Results is a list of pyomo
+    solver results objects.  Since this petsc DAE solver utility solves the
+    initial conditions first then may integrate over the time domain in one or
+    more steps, the results of multiple solves are returned.  The trajectory is
+    a PetscTrajectory object which gives the full trajectory of the solution
```

Comment:
Recommend adding back-ticks around ``PetscTrajectory`` to be consistent with the formatting used in the rest of the docstring